WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -2781,19 +2781,48 @@ extension AztecPostViewController {
 
     fileprivate func insert(exportableAsset: ExportableAsset, source: MediaSource, attachment: MediaAttachment? = nil) {
         var attachment = attachment
+
+        var externalMediaLinkObserver: MediaCoordinator.ObserverBlock?
+        var externalMediaLinkObserverReceipt: UUID?
+
         if attachment == nil {
             switch exportableAsset.assetMediaType {
             case .image:
                 attachment = insertImageAttachment()
             case .video:
                 attachment = insertVideoAttachmentWithPlaceholder()
             default:
-                return
+                guard let urlAsset = exportableAsset as? NSURL else { break }
```

Comment:
From @SergioEstevao's comment it sounds like some of this may need changing anyway, but whatever we keep here I'd probably recommend we extract it to a method, so this case becomes a single line like the `.image` and `.video` cases.